### ISSUE TITLE
App Crash on clicking Read aloud

### ISSUE THREAD

**Issue by XDivergent** [kiwix/kiwix-android#1414]
**Describe the bug**
App crash when clicked on Read aloud from the menu.

**Steps to reproduce the behaviour:**
1. Install new app
2. click + icon from sidebar
3. click Read aloud from menu.
4. crash

**Environment**
- Version of Kiwix Android : 2.5.3
- Device : redmi4
- OS version : Android

**Comment by XDivergent** [comment]
Can I help in fixing the bug?

**Comment by kelson42** [comment]
@XDivergent of course! Would be great! Waiting to your PR.

**Comment by XDivergent** [comment]
Sir, I need help in building kiwix-android

I am getting this error everytime I try to run `kiwix-build kiwix-android`
```
The target platform android cannot be build on host Darwin.
Select another target platform or change your host system.
```

**Comment by kelson42** [comment]
@XDivergent Have you read the `README.md` and `CONTRIBUTING.md` properly?

**Comment by jhbiggs** [comment]
@XDivergent could you please explain further how to reproduce this problem?  I have followed the steps on emulated Nexus API 28 and the reader succeeds.

**Comment by XDivergent** [comment]
I installed the app from playstore.

**Comment by abdulwd** [comment]
@XDivergent Can you build the app from the develop branch and check if the bug is present?

**Comment by XDivergent** [comment]
I am building the app but every time I am getting a web view app. Not the app available on play store.

**Comment by abdulwd** [comment]
Have you changed the build variant to `kiwixDebug`?
Also, the app on the play store is the older version. We have added many new features which are yet to be released.

**Comment by XDivergent** [comment]
> Have you changed the build variant to `kiwixDebug`?
> Also, the app on the play store is the older version. We have added many new features which are yet to be released.

Yes I am using kiwixDebug variant

**Comment by XDivergent** [comment]
@abdulwd Got it. my mistake, I misunderstood the new app as a web app. It looks like a browser but it is not😭. I did not find the option read aloud in this version.

**Comment by abdulwd** [comment]
Open a file then it will be there.

**Comment by abdulwd** [comment]
Have you downloaded any ZIM file?

**Comment by XDivergent** [comment]
> Have you downloaded any ZIM file?

Not showing any file for download in the online tab.

**Comment by abdulwd** [comment]
There must be a loading icon, the files are dynamically fetched so it takes some time. You have to wait after opening the online tab and don't close it.

**Comment by macgills** [comment]
Read Aloud is not present when no Zim File has been selected. Issue likely fixed in 3.0 some time ago.
Thank you for your report @XDivergent , closing
